### PR TITLE
Update configuring-teams-calling-quickstartguide.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
update internal doc

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks

File: Teams/configuring-teams-calling-quickstartguide.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The default configuration of TeamsUpgradePolicy is Islands mode, which is design
 To ensure that incoming VOIP and PSTN calls always land in a user's Teams client, update the user's coexistence mode to be TeamsOnly (which means, assign them the "UpgradeToTeams" instance of TeamsUpgradePolicy.  For more information on coexistence modes and TeamsUpgradePolicy, see [Migration and interoperability guidance for organizations using Teams together with Skype for Business](./migration-interop-guidance-for-teams-with-skype.md)
 
 **NOTES**
- - Skype for Business IP phones will receive calls, even if the user is in TeamsOnly mode.  
+ - Skype for Business IP phones will receive calls, even if the user is in TeamsOnly mode, if the IP phone has the mix mode aviliable
```
Comment (JohanFreelancer9) [inline Teams/configuring-teams-calling-quickstartguide.md:62]:
```suggestion
 - Skype for Business IP phones will receive calls, even if the user is in TeamsOnly mode, provided the IP phone has Mixed mode available.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions.

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by RuiPereiraTabares** [comment]
closing need to get full confirmation over ongoing incident

**Pull request closed without merging**